media/scss/_listset.scss

```diff
@@ -10,18 +10,6 @@
     &-listnum li {
         counter-increment: listNum-counter;
     }
-    &-listnum li::before {
```

Comment:
The removal of this shows weird CSS on my dev env

Reply:
I've restored the CSS and fixed the broken highlight.